packages/cli-repl/src/cli-repl.ts

```diff
@@ -45,6 +46,12 @@ class CliRepl {
       this.serviceProvider = serviceProvider;
       this.mapper = new Mapper(this.serviceProvider);
       this.shellApi = new ShellApi(this.mapper);
+      // .toString() output since child_process execution returns a Buffer
+      // grab version info from the first line from output
+      this.mdbVersion = execSync('mongod --version')
+        .toString()
+        .split('\n')[0]
+        .replace('db version ', '');
```

Comment:
this is hyper hacky ^ 
i am thinking there maybe should be a way for `shell-api` to provide us with a current mongod version.

Reply:
Technically we have to implement it anyway to do: `db.version()`

Reply:
ah good! this hack can then quickly be rewritten once we get that. 

i think with the node driver, we just need to get [serverInfo](https://mongodb.github.io/node-mongodb-native/3.5/api/Admin.html#serverInfo) and that will return all the info we will need

Reply:
Actually, didn't @durran implement the version command already? (the one you are calling).

Reply:
hmmmm not that i know! there is [this version](https://github.com/mongodb-js/mongosh/blob/master/packages/cli-repl/bin/mongosh.js#L12), but this looks at the version of `mongosh` package, rather than `mongod`

Reply:
Ohh `mongo version` returns the "shell version". Ok never mind. I think we can switch the hack to an hardcoded string.

Reply:
> I think we can switch the hack to an hardcoded string.

I don't understand what you mean 😅 `mongod version` returns the DB version for me:
![Capture d’écran, le 2020-02-06 à 16 03 39](https://user-images.githubusercontent.com/8107784/73949471-bd1c6500-48fa-11ea-8f92-c398267bce54.png)